ServerSide/yoty/Core/Managers/Bids/BidsManager.cs

```diff
@@ -36,17 +41,14 @@ public async Task<Response<BuyerDTO>> AddBuyer(BidBuyerJoinRequest bidBuyerJoinR
                 NumOfUnits = bidBuyerJoinRequest.Items
             });
             bid.UnitsCounter += bidBuyerJoinRequest.Items;
-            using (var new_context = new YotyContext())
+            try
+            {
+                _context.Bids.Update(bid);
+                await _context.SaveChangesAsync().ConfigureAwait(false);
+            }
+            catch (Exception ex)
             {
-                try
-                {
-                    new_context.Bids.Update(bid);
-                    await new_context.SaveChangesAsync().ConfigureAwait(false);
-                }
-                catch (Exception ex)
-                {
-                    return new Response<BuyerDTO>() { DTOObject = null, IsOperationSucceeded = false, SuccessOrFailureMessage = ex.Message };
-                }
+                return new Response<BuyerDTO>() { DTOObject = null, IsOperationSucceeded = false, SuccessOrFailureMessage = ex.Message };
```

Comment:
if we decide that we are not returning the buyertDTO then return Response() instead response<BuyerDTO>

Reply:
also change the message to be success message ....

Reply:
Will sync on the DTO dilema with Ofek and Yona

Reply:
What do you mean change the message to success? this is incase of exception, not success